pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -171,10 +173,8 @@ func (rc *reconciler) reconcile() {
 	// Ensure volumes that should be unmounted are unmounted.
 	for _, mountedVolume := range rc.actualStateOfWorld.GetMountedVolumes() {
 		if !rc.desiredStateOfWorld.PodExistsInVolume(mountedVolume.PodName, mountedVolume.VolumeName) {
-			// Volume is mounted, unmount it
-			glog.V(12).Infof(mountedVolume.GenerateMsgDetailed("Starting operationExecutor.UnmountVolume", ""))
-			err := rc.operationExecutor.UnmountVolume(
-				mountedVolume.MountedVolume, rc.actualStateOfWorld)
+			mountedVolume.VolumeHandler = operationexecutor.NewVolumeHandler(mountedVolume.VolumeSpec, rc.operationExecutor)
```

Comment:
If you're always creating the volume handler right before the operation, then there's no need to store it as a field in the volume struct.

Reply:
agree. I confirmed that handler are always created right before the operation, should be removed from volume struct.